dashboard/app/views/pd/workshop_enrollment/new.html.haml

```diff
@@ -3,88 +3,102 @@
 
 %div
   %h1
-    Register for a Code Studio workshop (for teachers)
+    Register for a Code Studio workshop
   %p
     Taught by Code.org facilitators who are experienced computer science educators,
     our workshops will prepare you to teach the Code.org curriculum.
 
-  %h2
-    Workshop Details
+  .container-fluid
+    .row-fluid
+      -# Left Column
+      .span6
+        .row-fluid
```

Comment:
I tried a number of things to get the 2 columns to work properly. I think the outer container is what did it. I can probably remove the -fluid parts.